functional-tests/src/test/groovy/FrontendCore/spec/Permit.groovy

```diff
@@ -11,17 +16,121 @@ import utils.*
 @Title("MDS-MineProfile-PermitTab")
 @Stepwise
 class  PermitSpec extends GebReportingSpec {
+    static PERMIT_NUMBER = "M-66666"
+
+    @Rule
+    TemporaryFolder dir = new TemporaryFolder()
+
+    @SourceURI
+    URI sourceUri
+    Path scriptLocation = Paths.get(sourceUri)
+
     def setup() {
         to MineProfilePage
+        permitTab.tabSelect.click()
     }
 
-    def "User can view permit"(){
-        when:"I go to the permit tab"
-        permitTab.tabSelect.click()
+    def "User can create a permit"(){
+        when: "I click on the new permit"
+        permitTab.newPermitButton.click()
+        newPermitForm.completePermitForm()
+        then: "A permit with the correct ID is present in the permit's tab"
+        permitTab.permitTitle.text() == PERMIT_NUMBER
+    }
+
+    def "User can edit the status of a permit"(){
+        when: "I hover over the the add/edit button."
+        moveToFooterAndHoverOnEdit()
+        permitTab.editPermitStatusButton.click()
+
+        and: "I change the status of the permit to closed."
+        // THIS NEEDS TO GET MOVED TO THE FORMS OPTIONS
```

Comment:
Might want to add a TODO or remove the comment :)

Reply:
done